packages/next/test/fixtures/00-legacy-mono-repo-server-mode/packages/nextjs/package.json

```diff
@@ -10,7 +10,7 @@
     "next": "canary",
     "react": "latest",
     "react-dom": "latest",
-    "typescript": "4.2.3"
+    "typescript": "4.9.5"
```

Comment:
```suggestion
    "typescript": "latest"
```
I think we're good to just use latest here so we don't need to keep bumping

Reply:
Agreed, we should use cron to update tests the same way we use cron to update examples

https://github.com/vercel/vercel/blob/main/.github/workflows/cron-update-next.yml

That would make it much more clear that a failure is due to a next/react/typescript update rather than a regression introduced in `@vercel/next`